### PR TITLE
Add Braille (Grade 1)

### PR DESCRIPTION
**Pull request by rogercyyu**
Hello, I added Braille grade 1 feature for [issue-439](https://github.com/Ciphey/Ciphey/issues/439).
Not sure how to do grade 2 aka contractions.

Not sure if I covered everything for grade 1...

Looking for feedback.

### CODE REVIEW

File: ciphey/basemods/Decoders/braille.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+from typing import Optional, Dict, List
+from loguru import logger
+from ciphey.iface import ParamSpec, Config, T, U, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class Braille(Decoder[T, U]):
+    UNICODE_STRING = "⠀⠁⠂⠃⠄⠅⠆⠇⠈⠉⠊⠋⠌⠍⠎⠏⠐⠑⠒⠓⠔⠕⠖⠗⠘⠙⠚⠛⠜⠝⠞⠟⠠⠡⠢⠣⠤⠥⠦⠧⠨⠩⠪⠫⠬⠭⠮⠯⠰⠱⠲⠳⠴⠵⠶⠷⠸⠹⠺⠻⠼⠽⠾⠿"
+    C_STRING = " a1b'k2l@cif/msp\"e3h9o6r^djg>ntq,*5<-u8v.%[$+x!&;:4\\0z7(_?w]#y)="
```
Comment (bee-san) [inline ciphey/basemods/Decoders/braille.py:9]: This should go into cipheydists, we have documentation for this here :) https://github.com/Ciphey/Ciphey/wiki/What-to-check-before-a-pull-request#dictionaries-or-lists

File: ciphey/basemods/Decoders/braille.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    translation = str.maketrans(UNICODE_STRING, C_STRING)
+
+    def decode(self, text: T) -> Optional[U]:
+        for c in text:
+            if str(c) in self.UNICODE_STRING:
+                continue
+            logger.trace(f"Non-Braille glyph '{c}' found")
```
Comment (bee-san) [inline ciphey/basemods/Decoders/braille.py:16]: This would be extremely spammy, please do not include this :D

**Review by bee-san — CHANGES_REQUESTED** [review]

**Comment by bee-san** [comment]
Requesting code review from @SkeletalDemise as they know far more about this kind of stuff than I do :P

File: ciphey/basemods/Decoders/braille.py (hunk 1 above)
Comment (rogercyyu) [reply ciphey/basemods/Decoders/braille.py:9]: Will do.

File: ciphey/basemods/Decoders/braille.py (hunk 2 above)
Comment (bee-san) [reply ciphey/basemods/Decoders/braille.py:16]: The logger.trace

**Comment by rogercyyu** [comment]
Hey so, I am gonna let @Trisanu-007 worked on these issue on my branch for a bit (till Oct 8th), if that is okay with the team? Or can GitHub allow 2 people on a PR?

**Comment by bee-san** [comment]
> Hey so, I am gonna let @Trisanu-007 worked on these issue on my branch for a bit (till Oct 8th), if that is okay with the team? Or can GitHub allow 2 people on a PR?

Hey! I can mark one of you as a co-author :)

**Comment by rogercyyu** [comment]
That would be awesome! Thank-you! :D

File: ciphey/basemods/Decoders/braille.py (hunk 1 above)
Comment (Trisanu-007) [reply ciphey/basemods/Decoders/braille.py:9]: while moving this dictionary to cipheydists should we create a pull request?

**Comment by bee-san** [comment]
CipheyDists guide:
https://github.com/Ciphey/Ciphey/wiki/What-to-check-before-a-pull-request#dictionaries-or-lists

**Comment by bee-san** [comment]
Hey, you need to add the alphabet to cipheydists :)

**Comment by rogercyyu** [comment]
Hey, slowly working on it... I'll send a PR to cipheydists when I feel I got most of the symbols ready :P

**Comment by SkeletalDemise** [comment]
@all-contributors please add @rogercyyu for code

**Review by SkeletalDemise — APPROVED** [review]

**Pull request merged**